pkg/monitoring/metrics/forklift-controller/metrics.go

```diff
@@ -0,0 +1,117 @@
+package forklift_controller
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+const (
+	Succeeded = "Succeeded"
+	Failed    = "Failed"
+	Executing = "Executing"
+	Running   = "Running"
+	Pending   = "Pending"
+	Canceled  = "Canceled"
+	Blocked   = "Blocked"
+	Ready     = "Ready"
+	Deleted   = "Deleted"
+	Warm      = "Warm"
+	Cold      = "Cold"
+	Local     = "Local"
+	Remote    = "Remote"
+)
+
+var (
+	// 'status' - [ Succeeded, Failed, Executing, Canceled]
+	// 'provider' - [oVirt, VSphere, Openstack, OVA, Openshift]
+	// 'mode' - [Cold, Warm]
+	// 'target' - [Local, Remote]
+	migrationStatusGauge = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "mtv_migrations_status",
+		Help: "VM Migrations sorted by status, provider, mode and destination",
+	},
+		[]string{
+			"status",
+			"provider",
+			"mode",
+			"target",
+		},
+	)
+
+	// 'status' - [ Succeeded, Failed, Executing, Running, Pending, Canceled, Blocked, Deleted]
+	// 'provider' - [oVirt, VSphere, Openstack, OVA, Openshift]
+	// 'mode' - [Cold, Warm]
+	// 'target' - [Local, Remote]
+	planStatusGauge = promauto.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
Do you still need this metric?

Reply:
If you do and its per 1 `Plan` I think the name should be  `mtv_plan_status`

Reply:
Yeah, we talked about it. We decided to leave it even at the cluster level. It also reflects many more statuses that migration doesn't have, so it can be useful for understanding certain behaviors and even for displaying information in the UI console.